File: 백준/Bronze/15552. 빠른 A＋B/빠른 A＋B.py
import sys  

input = sys.stdin.read 
data = input().splitlines() 

num = int(data[0])
result = []

for i in range(1, num + 1): 
    a, b = map(int, data[i].split()) 
    result.append(str(a + b)) 

sys.stdout.write("\n".join(result) + "\n") 

File: 백준/Bronze/15552. 빠른 A＋B/빠른 A＋B.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

let testCase = Number(input[0]);
let answer = '';

for (let t = 1; t <= testCase; t++) {
    let [a, b] = input[t].split(' ').map(Number);
    answer += (a + b) + '\n';
}

console.log(answer);